### PR TITLE
Trunk-5650:Add parent order group to order group

### PR DESCRIPTION
**Pull request by gitcliff**
link:https://issues.openmrs.org/browse/TRUNK-5650
Add parent order group to order group

### CODE REVIEW

**Comment by gitcliff** [comment]
@dkayiwa this is s new pr with resolved conflicts

File: api/src/main/java/org/openmrs/OrderGroup.java [hunk 1]
```diff
@@ -208,4 +209,23 @@ public void setId(Integer id) {
 		setOrderGroupId(id);
 	}
 	
+	/**
+	 * Gets the parent order group to maintain linkages between groups and support group nesting
+	 * 
+	 * @param returns the parent order group
+	 * @since 2.3.0..
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/OrderGroup.java]: Because 2.3.0 is already cut, this better be changed to 2.4.0

**Comment by dkayiwa** [comment]
Have you tried to run the openmrs web application with these changes?

**Comment by gitcliff** [comment]
yes @dkayiwa  and i attached the screenshots

File: api/src/main/java/org/openmrs/OrderGroup.java [hunk 2]
```diff
@@ -208,4 +209,23 @@ public void setId(Integer id) {
 		setOrderGroupId(id);
 	}
 	
+	/**
+	 * Gets the parent order group to maintain linkages between groups and support group nesting
+	 * 
+	 * @param returns the parent order group
+	 * @since 2.4.0..
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/OrderGroup.java]: What is the use of the two .. after 0?

Comment (gitcliff) [reply api/src/main/java/org/openmrs/OrderGroup.java]: i have fixed it

**Pull request merged**